### ISSUE TITLE
Autocomplete

### ISSUE THREAD

**Issue by 0zguner** [fimkap/vimrc#1]
Hi! I've watched your youtube videos. I was wondering if your nvimrc file enables autocomplete for swift/ios projects ?

**Comment by fimkap** [comment]
Hi! It is not yet in my nvimrc. The best autocompletion for Swift that I found is https://github.com/maralla/completor-swift and https://github.com/maralla/completor.vim. However, for me it worked only for Vim8. I need to enable some extra features in Neovim for it to work. When I started, the completor for Swift had only basic completion after the dot. I started to play with my local version to change it and then talked to the author and he fixed the completor for Swift to complete from any place where the SourceKit can work. Also he added placeholders. I still had some local changes to make it work in some edge cases. So it looked quite promising at that stage. The most important part for the completor is that it has a fast source and the completions are fast, contrary to the deoplete swift sources I tried. I will check the current state of the completor for Swift 3 and update my configuration.
